### ISSUE TITLE
Unsupported species for H.sapiens ensembl 92

### ISSUE THREAD

**Issue by zlskidmore** [murphycj/AGFusion#25]
Hi, I'm getting an unsupported species error. Looking at the docs and code this appears to be referencing the pyensembl database correct? I'm using the latest release 1.2 with ensembl version 92. Any suggestions would be appreciated!

Zach

### Error

```
Traceback (most recent call last):
  File "/usr/local/bin/agfusion", line 5, in <module>
    cli.main()
  File "/usr/local/lib/python3.6/dist-packages/agfusion/cli.py", line 535, in main
    assert species in AVAILABLE_ENSEMBL_SPECIES, 'unsupported species!'
AssertionError: unsupported species!
```

### Command

```
agfusion batch -f star-fusion.fusion_predictions.tsv -a starfusion -db /opt/pyensembl/pyensembl/GRCh38/ensembl92/Homo_sapiens.GRCh38.92.gtf.db -o agfusion
```

**Comment by murphycj** [comment]
Zach, the -db flag is for the agfusion prebuilt database, which you can download with “agfusion download” command. Enter “agfusion download -h” for more information. 

Let me know if that helps!

**Comment by zlskidmore** [comment]
yes that was it, thanks for the help! Looks like release 1.2 might not support ensembl 92 as well but rolling that back to 87 worked

**Comment by murphycj** [comment]
great that you got it working. Actually, there is a more recent version of agfusion available on python pip (v1.24). You can run “pip install agfusion --upgrade”. I haven’t made the release on GitHub yet. It supports Ensembl 92. You can also try that if you want.
